### DISCUSSION TITLE
Error running test cases on Windows

### DISCUSSION THREAD

**Discussion by vincentdo1** [radicalplatforms/shafa#114] [Q&A] [answered]
This is the error I get after installing packages and postgres 16. Was wondering if there is a work around for windows.
![image](https://github.com/radicalplatforms/shafa/assets/85699604/c337fc04-8f7a-4369-886b-c441eb7fe3c3)

**Comment by rak3rman** [comment, marked as answer]
Ah, this is (partially) intended behavior of the test utility that provisions the Postgres database locally. In this case, I wasn't able to test these scripts against a Windows machine (I use Mac and Ubuntu for development) so the script intentionally throws a fatal error if it detects a Windows environment.

Running on Windows will require an addition to the `getInstallationScript()` function located in `shafa/hono/test/utils/db.ts`. I'll go ahead and open a new issue requesting this implementation.

https://github.com/radicalplatforms/shafa/blob/main/hono/test/utils/db.ts#L98

**Comment by rak3rman** [comment]
Issue will now be tracked in issue #115. Closing discussion.
